src/rust/ensogl/src/display/symbol/gpu.rs

```diff
@@ -348,6 +348,12 @@ impl Symbol {
                 let count          = self.surface.point_scope().size()    as i32;
                 let instance_count = self.surface.instance_scope().size() as i32;
 
+                debug_assert_eq!(
+                    self.context.check_framebuffer_status(Context::FRAMEBUFFER),
+                    Context::FRAMEBUFFER_COMPLETE,
+                    "Framebuffer does not have status FRAMEBUFFER_COMPLETE."
+                );
+
```

Comment:
Super cool! One request - please make the error more desriptive. If someone gets this error it should be understandable for us what happened there.